### PR TITLE
feat: encode created meshes in draco format

### PR DESCRIPTION
**Pull request by manuel-castro**
Adds DracoPy as a dependency. Opening this again now.

### CODE REVIEW

**Comment by william-silversmith** [comment]
This is very cool! What changed from the last PR? It was a little confusing.

**Comment by manuel-castro** [comment]
Oh sorry about that. I had an option in DracoPy that allowed users to encode encoding options such as the number of bits used for quantization to the beginning of their file. I removed this in DracoPy (@nkemnitz pointed out to me I can encode this information within draco as metadata instead -- much better) and so I removed passing in the option from igneous too

File: igneous/tasks.py [hunk 1]
```diff
@@ -264,7 +265,14 @@ def __init__(self, shape, offset, layer_path, **kwargs):
         'low_padding': kwargs.get('low_padding', 0),
         'high_padding': kwargs.get('high_padding', 1),
         'parallel_download': kwargs.get('parallel_download', 1),
-        'cache_control': kwargs.get('cache_control', None)
+        'cache_control': kwargs.get('cache_control', None),
+        'use_draco_encoding': kwargs.get('use_draco_encoding', False),
```
Comment (william-silversmith) [inline igneous/tasks.py]: It might be more backwards compatible to write `'encoding': 'draco'` in case we ever add more options.

Comment (manuel-castro) [reply igneous/tasks.py]: Agreed

File: igneous/tasks.py [hunk 2]
```diff
@@ -266,14 +266,21 @@ def __init__(self, shape, offset, layer_path, **kwargs):
...
     }
     self.draco_encoding_settings = {
       'quantization_bits': kwargs.get('draco_quantization_bits', 14),
       'compression_level': kwargs.get('draco_compression_level', 1),
       'quantization_range': kwargs.get('draco_quantization_range', -1),
       'quantization_origin': kwargs.get('draco_quantization_origin', None)
     }
+    supported_encodings = ['precomputed', 'draco']
+    if not self.options['encoding_type'] in supported_encodings:
+      raise ValueError('Encoding type specified is not supported')
```
Comment (william-silversmith) [inline igneous/tasks.py]: The encoding type should be shown in the error message, otherwise you might drive yourself crazy debugging. :sob:

**Comment by william-silversmith** [comment]
Let's run this on a test dataset. I'm building a docker `seunglab/igneous:draco`

**Comment by william-silversmith** [comment]
Medium dataset test looks good.

**Pull request merged**